samples/client.c

```diff
@@ -868,13 +921,15 @@ migrate_from(int sock, size_t *nr_iters, struct iovec **migr_iters,
         .crcp = crcp
     };
 
+    size_t max_iter_size = 4096;
```

Comment:
get it from `sysconf(_SC_PAGE_SIZE)`

Reply:
This was actually incorrect behaviour - I just set to the page size because I didn't realise that what we actually want is the maximum supported transfer size, as determined by the negotiation when we connect. In the client example, `CLIENT_MAX_DATA_XFER_SIZE` is 1024, which allows us to test the chunking.

Reply:
That was for DMA, not sure how it affects migration, can you investigate?